velodyne_pointcloud/src/lib/rawdata.cc

```diff
@@ -255,6 +290,11 @@ inline float SQR(float val) { return val*val; }
       cos_rot_table_[rot_index] = cosf(rotation);
       sin_rot_table_[rot_index] = sinf(rotation);
     }
+
+    for (uint8_t i = 0; i < 16; i++) {
+      vls_128_laser_azimuth_cache[i] = (VLS128_CHANNEL_TDURATION / VLS128_SEQ_TDURATION) * (i + i / 8);
+    }
```

Comment:
Can you please refactor `RawData::setup` and `RawData::setupOffline` so they call a shared function? There is a lot of duplicated code between these two.

Reply:
Thanks for this! Much improved!